localisation/00_wc_sad.csv

```diff
@@ -773,3 +773,6 @@ whispers_option_stewardship;
 whispers_option_intrigue;�Y[Root.GetFirstName]�! will focus on writing down whispers about �YIntrigue�!.;;;;;;;;;;;;;x
 whispers_option_learning;�Y[Root.GetFirstName]�! will focus on writing down whispers about �YLearning�!.;;;;;;;;;;;;;x
 nick_the_corruptor;the Corruptor;;[From.Getderdie] Verderber[From.GetinFemale];;;;;;;;;;;x
+text_effect_qiraji_retinue;Enables �YSilithid Swarm�! retinue avaible to �YQiraji�! culture.;;Aktiviert das �YSilithiden-Schwarm�!-Gefolge f�r die �YQiraji�!-Kultur.;;;;;;;;;;;x
+text_effect_naga_retinue;Enables �YRoyal Guard�! retinue avaible to �YNaga�! culture.;;Aktiviert das �YK�nigswache�!-Gefolge f�r die �YNaga�!-Kultur.;;;;;;;;;;;x
+text_effect_titanforged_retinue;Enables �YTitan-forged Legion�! retinue avaible to �YTitan-forged�! culture group.;;Aktiviert das �YTitanische Legion!-Gefolge f�r die �Ytitanische�! Kultur.;;;;;;;;;;;x
```

Comment:
Typo in available. Also, it sounds like you must have these cultures to unlock them.